lists/task_02.py

```diff
@@ -0,0 +1,10 @@
+from random import randint
+
+n = 100
+quantity = 10
+worked_list = [0] * quantity
+
+for i in range(quantity):
+    worked_list[i] = randint(0, n)
+worked_list.sort()
```

Comment:
everything was solved correctly, but let's do the sorting manually by comparing neighboring elements